### PR TITLE
Added iphone and ipad profiles to cucumber.yml for different running configurations for the ruby cucumber sample

### PR DESCRIPTION
**Pull request by faidy**
- Added iphonesim and ipadsim profiles to cucumber.yml
- Defined IDEVICENAME environment variable in cucumber.yml
- Used IDEVICENAME in env.rb to load the appium.txt file related to the configuration
- Left the regular configuration as is so as not to cause any confusion
- Now, running `cucumber -p ipadsim` will launch the ipad air simulator while `cucumber -p iphonesim` will launch the iphone 6 simulator

### CODE REVIEW

File: sample-code/examples/ruby/cucumber_ios/config/cucumber.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# config/cucumber.yml
+##YAML Template
+---
+ipadsim: IDEVICE=ipad IDEVICENAME='ipad simulator'
```
Comment (bootstraponline) [inline sample-code/examples/ruby/cucumber_ios/config/cucumber.yml]: Is IDEVICE used? In env.rb, I only see IDEVICENAME.

Comment (faidy) [inline sample-code/examples/ruby/cucumber_ios/config/cucumber.yml]: I intended to use it in step definitions that are different on different form factors but I didn't find an applicable case here.
I will remove it now.

**Comment by faidy** [comment]
@bootstraponline I removed the unused variable, kindly check and let me know your feedback.

**Comment by bootstraponline** [comment]
Looks good. Please squash to one commit, then comment once the PR is updated (notifications aren't sent by github unless a comment is added).

```
git reset --soft head~5
git commit -am "Read configs from cucumber.yml"
git push -f
```

**Comment by faidy** [comment]
Thanks @bootstraponline, Done as requested

**Comment by bootstraponline** [comment]
It looks like there are quite a few changes here that weren't there before?

- The two apk files are already in the repo, not sure why they'd be added again.
- Are the java changes intentional?

**Comment by faidy** [comment]
I think they are a result of updating the fork, check pull requests #33  and #34

**Comment by bootstraponline** [comment]
No, I'm pretty sure this PR isn't rebased properly which is why the extra changes are showing up. I've submitted #39 based on this one. If that contains all the desired changes, let me know and I'll merge. It retains your author info.

**Comment by faidy** [comment]
Great, yes these are the desired ones to be merged.

**Comment by bootstraponline** [comment]
Great, thanks for the PR! It's been merged. In the future, the `git pull --rebase` command is usually the solution for weird changes showing up.

**Comment by faidy** [comment]
Thanks for the hint.

**Pull request closed without merging**